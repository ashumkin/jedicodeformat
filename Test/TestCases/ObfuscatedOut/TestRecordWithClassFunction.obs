unit testrecordwithclassfunction;interface type ttestrecord=record private _lo:uint64
;_hi:int64;public class function create(low:uint64;high:int64):ttestrecord;static
;end;implementation class function ttestrecord.create(low:uint64;high:int64):ttestrecord
;begin result._lo:=low;result._hi:=high;end;end. 