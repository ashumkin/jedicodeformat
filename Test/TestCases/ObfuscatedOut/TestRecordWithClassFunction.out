unit testrecordwithclassfunction;

interface

type
  ttestrecord = record
  private
    _lo: uint64;
    _hi: int64;
  public
    class function Create(low: uint64; high: int64): ttestrecord; static;
  end;

implementation

class function ttestrecord.Create(low: uint64; high: int64): ttestrecord;
begin
  Result._lo := low;
  Result._hi := high;
end;

end.
 